examples/plotting/hmi_synoptic_maps.py

```diff
@@ -1,12 +1,9 @@
 """
-==============================================
-Loading an HMI daily synoptic image into a Map
-==============================================
-
-In this example we load the Daily Synoptic Maps produced by the HMI team. This
-data is an interesting demonstration of sunpy's Map class as it is not in the
-more common Helioprojective coordinate system, it is in Heliographic Carrington
-coordinates and in a non-trivial Cylindrical Equal Area projection.
```

Comment:
I feel like this provides some more context to the example?

Reply:
I didn't think it was that relevant any more as `Map` works out of the box for different projections. Happy to put it back though?

Reply:
That's fair but I think leaving the example main text as `In this example we load a synoptic map produced by the HMI team.` is a bit lacking on information or context.

Reply:
I put this back in, with some minor modifications.